docs/ads.md

```diff
@@ -0,0 +1,458 @@
+
+#
+
+> ### <a name="ALL_ADS_COMPLETED"></a>ALL_ADS_COMPLETED
+>
+> Fires when the player is done playing all the ads.
+
+#
+
+> ### <a name="AD_BREAK_START"></a>AD_BREAK_START
+>
+> Fires before an ad break is about to start.
+> <br><br>_payload parameters:_
+>
+> | Name      | Type                  | Description        |
+> | --------- | --------------------- | ------------------ |
+> | `adBreaK` | [`AdBreak`](#AdBreak) | The ad break data |
+
+#
+
+> ### <a name="AD_BREAK_END"></a>AD_BREAK_END
+>
+> Fires when an ad break is finished.
+
+#
+
+> ### <a name="AD_FIRST_QUARTILE"></a>AD_FIRST_QUARTILE
+>
+> Fires when the ad playhead crosses first the quartile.
+
+#
+
+> ### <a name="AD_MIDPOINT"></a>AD_MIDPOINT
+>
+> Fires when the ad playhead crosses the midpoint.
+
+#
+
+> ### <a name="AD_THIRD_QUARTILE"></a>AD_THIRD_QUARTILE
+>
+> Fires when the ad playhead crosses the third quartile.
+
+#
+
+> ### <a name="USER_CLOSED_AD"></a>USER_CLOSED_AD
+>
+> Fires when the ad is closed by the user.
+
+#
+
+> ### <a name="AD_VOLUME_CHANGED"></a>AD_VOLUME_CHANGED
+>
+> Fires when the ad volume has changed.
+
+#
+
+> ### <a name="AD_MUTED"></a>AD_MUTED
+>
+> Fires when the ad volume has been muted.
+
+#
+
+> ### <a name="AD_PROGRESS"></a>AD_PROGRESS
+>
+> Fires on ad time progress.
+> <br><br>_payload parameters:_
+>
+> | Name         | Type               | Description           |
+> | ------------ | ------------------ | --------------------- |
+> | `adProgress` | `AdProgressObject` | The ad progress data |
+>
+> ###### Type `AdProgressObject`
+>
+> ```js
+> {
+>   currentTime: number,
+>   duration: number
+> }
+> ```
+
+## Ads API Reference
+
+- [AdBreak][1]
+  - [Parameters][2]
+  - [type][3]
+  - [position][4]
+  - [numAds][5]
+- [Ad][6]
+  - [Parameters][7]
+  - [id][8]
+  - [contentType][9]
+  - [url][10]
+  - [title][11]
+  - [position][12]
+  - [duration][13]
+  - [clickThroughUrl][14]
+  - [posterUrl][15]
+  - [skipOffset][16]
+  - [linear][17]
+  - [width][18]
+  - [height][19]
+  - [bitrate][20]
+  - [skippable][21]
+- [AdsController][22]
+  - [Parameters][23]
+  - [allAdsCompleted][24]
+  - [isAdBreak][25]
+  - [getAdBreaksLayout][26]
+  - [getAdBreak][27]
+  - [getAd][28]
+  - [skipAd][29]
+  - [playAdNow][30]
+    - [Parameters][31]
+
+## <a name="AdBreak"></a>AdBreak
+
+### Parameters
+
+- `options` **PKAdBreakOptions** - The ad break data options.
+
+### type
+
+Returns **[string][32]** - The ad break type - pre/mid/post.
+
+### position
+
+Returns **[string][32]** - The ad break position on the playback timeline.
+
+### numAds
+
+Returns **[string][32]** - The number of ads inside the ad break.
+
+## <a name="Ad"></a>Ad
+
+### Parameters
+
+- `id` **[string][32]** Ad ID.
+- `options` **PKAdOptions** - The ad data options.
+
+### id
+
+Returns **[string][32]** - The ad ID.
+
+### contentType
+
+Returns **[string][32]** - The ad content type.
```

Comment:
This description originally said "The ad ID" but that's wrong, right? just verifying...